### ISSUE TITLE
Construction rights are not displayed

### ISSUE THREAD

**Issue by SidaRadis** [OpenRCT2/OpenRCT2#5741]
OS: Windows 7
Version: 0.0.8
Commit/Build: 659ccc0

The little icons that show available construction rights are not displayed. (if you move your mouse over where they usually can be bought the money it would cost you is still visible in the land rights window)
If you click construction rights and then go back to buy land, the buy land icons also disappear.

Screenshot:
![pokey park 2017-07-01 14-32-49](https://user-images.githubusercontent.com/28242696/27764564-772b8058-5e6a-11e7-80d8-b587a7cccd63.png)

**Comment by Fisherpon** [comment]
This is happening in my copy as well, funny enough on the same map. I also checked Bumbly Beach and saw the invisble flags were happening there too, though I couldn't double-check the disappearance of the buy flags afterwards as there is no land to buy on Bumbly Beach.

OS: Windows 10
Version: 0.1.1
Commit/Build: ca1aaec

**Comment by Gymnasiast** [comment]
Can confirm.

**Comment by SidaRadis** [comment]
I think #6178 tried to fix this issue but it still occurs after switching between "buy land" and "buy construction rights"

**Comment by nccvoyager** [comment]
I noticed this the other day, myself.
In RCT2, there aren't that many base scenarios that can allow this issue to occur.
(If a scenario doesn't have land to buy, or construction rights to buy, one button is grayed out, and this issue cannot occur.)
The RCT2 "Dusty Greens" scenario appears to be the first base RCT2 scenario that this can occur in.

I noticed a few things about this. (Last one is the most notable to me.)
This issue occurs even if the "Land rights" window is closed before switching away from the "Construction rights" view.
Reloading the scenario will fix this graphical issue. (Save, load, continue playing.)
Land and construction rights can still be bought in spite of not being visually shown.

Still occurs as of:
Version 0.1.2 - develop - e1568aa
